python/lsst/ts/m2gui/display/figure.py

```diff
@@ -0,0 +1,167 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Figure"]
+
+import numpy as np
+
+from PySide2.QtCore import Qt, QPointF
+from PySide2.QtCharts import QtCharts
+from PySide2.QtGui import QPainter
+
+
+class Figure(QtCharts.QChartView):
```

Comment:
The naming creates the impression it's a generic figure. It's not, as you cannot change a number of data points. Renaming that to ConstantFigure/FigureConstant or ActuatorFigure/FigureActuator shall not do any harm.

Reply:
Renamed in the commit: [2c12f29](https://github.com/lsst-ts/ts_m2gui/pull/13/commits/2c12f2944ce244d168c829496b6b258554a4beae). Thanks!